src/grt/include/grt/GlobalRouter.h

```diff
@@ -395,6 +397,10 @@ class GlobalRouter
   float caps_perturbation_percentage_;
   int perturbation_amount_;
 
+  // variables congestion report file
+  bool save_congestion_;
+  const char * congestion_file_name_;
```

Comment:
save_congestion_ is just congestion_file_name_ != nullptr so I don't think you need it.